cloudify_agent/operations.py

```diff
@@ -480,7 +481,7 @@ def validate_agent_amqp(current_amqp=True, manager_ip=None,
 def transfer_agent_amqp(transfer_agent_timeout=300,
                         manager_ip=None, manager_certificate=None,
                         manager_rest_token=None, **_):
-
+    manager_ip = _get_network_ip(manager_ip)
```

Comment:
And if you've already validated the agent, and you have the dict, pass it to the function, instead of using runtime props again